engine/src/main/java/org/terasology/logic/console/ConsoleSystem.java

```diff
@@ -24,22 +25,39 @@
 import org.terasology.input.ButtonState;
 import org.terasology.input.binds.general.ConsoleButton;
 import org.terasology.logic.console.commandSystem.ConsoleCommand;
+import org.terasology.logic.console.ui.MiniChatOverlay;
 import org.terasology.network.ClientComponent;
 import org.terasology.registry.In;
 import org.terasology.rendering.nui.NUIManager;
 
 @RegisterSystem
 public class ConsoleSystem extends BaseComponentSystem {
+
+    private static final ResourceUrn MINICHAT_UI = new ResourceUrn("engine:minichatOverlay");
```

Comment:
That's an interesting idea - I like it. The word "chat" could be confusing though. Maybe we can find a better name... `NotificationOverlay` maybe? Does not need to happen in this PR though.